packages/xo-web/src/common/intl/messages.js

```diff
@@ -860,8 +860,8 @@ const messages = {
   srRescan: 'Rescan all disks',
   srReconnectAll: 'Connect to all hosts',
   srDisconnectAll: 'Disconnect from all hosts',
-  srForget: 'Forget this SR',
-  srsForget: 'Forget SRs',
+  srForget: 'Forget SRs',
```

Comment:
This changes the meaning of the message.
I suggest we keep `srForget` and `srsForget` as they were and we create `nSrsForget` with values.